src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -138,4 +157,17 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPersonList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the book at the given {@code targetIndex} in the
+     * {@code model}'s address book.
+     */
+    public static void showBookAtIndex(Model model, Index targetIndex) {
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredBookList().size());
+
+        Book book = model.getFilteredBookList().get(targetIndex.getZeroBased());
+        model.updateFilteredBookList(x -> x.equals(book));
+
+        assertEquals(1, model.getFilteredBookList().size());
+    }
```

Comment:
I have written this function as well, will resolve merge conflicts on my end as well.